static/admin/config.yml

```diff
@@ -22,4 +25,82 @@ collections:
+    folder: "docs/guides"
+    create: true
+    slug: "{{slug}}"
+    preview_path: "docs/guides/{{slug}}"
+    editor:
+      preview: false
+    fields:
+      - {label: "Title", name: "title", widget: "string"}
+      - {label: "Description", name: "description", widget: "string", required: false, hint: "The description of the post which will be used by search engines and link previews. If not present, will default to the first line of the post contents."}
+      - {label: "Cover Image",name: "image", widget: "image", required: false,  hint: "Cover or thumbnail image that will be used when displaying the link to the post."}
+      - {label: "Body", name: "body", widget: "markdown"}
+  - name: "guides-platforms"
+    label: "Guides > Platforms"
+    folder: "docs/guides/platforms"
+    create: true
+    slug: "{{slug}}"
+    preview_path: "docs/guides/platforms/{{slug}}"
+    editor:
+      preview: false
+    fields:
+      - {label: "Title", name: "title", widget: "string"}
+      - {label: "Description", name: "description", widget: "string", required: false, hint: "The description of the post which will be used by search engines and link previews. If not present, will default to the first line of the post contents."}
+      - {label: "Cover Image",name: "image", widget: "image", required: false,  hint: "Cover or thumbnail image that will be used when displaying the link to the post."}
+      - {label: "Body", name: "body", widget: "markdown"}
+  - name: "setup"
+    label: "Setup and Usage"
+    folder: "docs/setup"
+    create: true
+    slug: "{{slug}}"
+    preview_path: "docs/setup/{{slug}}"
+    editor:
+      preview: false
+    fields:
+      - {label: "Title", name: "title", widget: "string"}
+      - {label: "Description", name: "description", widget: "string", required: false, hint: "The description of the post which will be used by search engines and link previews. If not present, will default to the first line of the post contents."}
+      - {label: "Cover Image",name: "image", widget: "image", required: false,  hint: "Cover or thumbnail image that will be used when displaying the link to the post."}
+      - {label: "Body", name: "body", widget: "markdown"}
+  - name: "setup-plugins"
+    label: "Setup and Usage > Plugins"
+    folder: "docs/setup/plugins"
+    create: true
+    slug: "{{slug}}"
+    preview_path: "docs/setup/plugins/{{slug}}"
+    editor:
+      preview: false
+    fields:
+      - {label: "Title", name: "title", widget: "string"}
+      - {label: "Description", name: "description", widget: "string", required: false, hint: "The description of the post which will be used by search engines and link previews. If not present, will default to the first line of the post contents."}
+      - {label: "Cover Image",name: "image", widget: "image", required: false,  hint: "Cover or thumbnail image that will be used when displaying the link to the post."}
+      - {label: "Body", name: "body", widget: "markdown"}
```

Comment:
was this autogenerated or added by hand?

Reply:
Added by hand